### ISSUE TITLE
issue on macOS with "linked" python2

### ISSUE THREAD

**Issue by pepper-jk** [tklab-tud/ID2T#102]
```
./build.sh: line 34: python3: command not found

Error: Could not create the venv. Please make sure the 'venv' Python-module is installed.

/Users/travis/.travis/functions: line 110: ./run_tests: No such file or directory
```

The following macOS and xcode combinations are known the make problems regarding python 3.8:

[macOS 10.14 with xcode11](https://travis-ci.org/tklab-tud/ID2T/jobs/635211625)
[macOS 10.13 with xcode10.1](https://travis-ci.org/tklab-tud/ID2T/jobs/635211626)

[travisCI ref](https://docs.travis-ci.com/user/reference/osx)

I'm assuming some problem with homebrew and the python binary, they probably changed it back to just "python".

I'll look into it when I have time to set up a new macOS VM.

**Comment by pepper-jk** [comment]
I could not reproduce this issue in my macOS VM.

However It seems to be a problem with the brew installation though.
I looked a bit more closely at the travisCI log and found this:

```
==> Upgrading python 
==> Downloading https://homebrew.bintray.com/bottles/python-3.7.6_1.high_sierra.
==> Downloading from https://akamai.bintray.com/35/3504f29ae0366d08fc025cfe2885c
######################################################################## 100.0%
==> Pouring python-3.7.6_1.high_sierra.bottle.tar.gz
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /usr/local
Could not symlink Frameworks/Python.framework/Headers
Target /usr/local/Frameworks/Python.framework/Headers
is a symlink belonging to python@2. You can unlink it:
  brew unlink python@2
To force the link and overwrite all conflicting files:
  brew link --overwrite python
To list all files that would be deleted:
  brew link --overwrite --dry-run python
Possible conflicting files are:
/usr/local/Frameworks/Python.framework/Headers -> /usr/local/Cellar/python@2/2.7.17/Frameworks/Python.framework/Headers
/usr/local/Frameworks/Python.framework/Python -> /usr/local/Cellar/python@2/2.7.17/Frameworks/Python.framework/Python
/usr/local/Frameworks/Python.framework/Resources -> /usr/local/Cellar/python@2/2.7.17/Frameworks/Python.framework/Resources
/usr/local/Frameworks/Python.framework/Versions/Current -> /usr/local/Cellar/python@2/2.7.17/Frameworks/Python.framework/Versions/Current
==> /usr/local/Cellar/python/3.7.6_1/bin/python3 -s setup.py --no-user-cfg insta
==> /usr/local/Cellar/python/3.7.6_1/bin/python3 -s setup.py --no-user-cfg insta
==> /usr/local/Cellar/python/3.7.6_1/bin/python3 -s setup.py --no-user-cfg insta
==> Caveats
Python has been installed as
  /usr/local/bin/python3
Unversioned symlinks `python`, `python-config`, `pip` etc. pointing to
`python3`, `python3-config`, `pip3` etc., respectively, have been installed into
  /usr/local/opt/python/libexec/bin
If you need Homebrew's Python 2.7 run
  brew install python@2
You can install Python packages with
  pip3 install <package>
They will install into the site-package directory
  /usr/local/lib/python3.7/site-packages
See: https://docs.brew.sh/Homebrew-and-Python
```

**Comment by pepper-jk** [comment]
Fixed with dc428314f4a032902660e517c6ed4c88d8269c44.
